<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPhrase</class>
 <widget class="QDialog" name="AddPhrase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>371</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>371</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>新建短语库</string>
  </property>
  <widget class="QLabel" name="label_tip">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>79</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>短语名称:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_phrase_name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>271</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_ok">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_cancel">
   <property name="geometry">
    <rect>
     <x>281</x>
     <y>450</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>369</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QPushButton" name="close_push_button">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="title_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>编辑短语</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="phrase_name_tip_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>说明：最长15个字符，支持半角的英文、数字
、下划线和冒号。英文开头，大小写敏感，
数字前必须要有下划线。</string>
   </property>
  </widget>
  <widget class="QLabel" name="display_position_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>显示位置：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="display_position_comboBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="content_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>67</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>内容：</string>
   </property>
  </widget>
  <widget class="CustomizeQTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>331</width>
     <height>100</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_tip_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>200</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>说明：最多100个字符</string>
   </property>
  </widget>
  <widget class="QLabel" name="line_label_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>5</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="line_label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>393</y>
     <width>5</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomizeQTextEdit</class>
   <extends>QTextEdit</extends>
   <header location="global">customize_ui/customizeqtextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
